Assets/HoloToolkit/SpatialSound/Scripts/AudioEmitter.cs

```diff
@@ -1,139 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using UnityEngine;
-
-namespace HoloToolkit.Unity
-{
-    /// <summary>
-    /// Class which supports IAudioInfluencers being used with audio sources.
-    /// </summary>
-    [RequireComponent(typeof(AudioSource))]    
-    public class AudioEmitter : MonoBehaviour 
```

Comment:
Cannot just rename classes when the intention for the script has not changed.  Revert this change and mark the class as [Obsolete("AudioEmitter is being deprecated, please use AudioInfluencerManager instead")].  This causes a braking change which should be avoided for Master. (fine for dev_working)

What is the reason for the file rename?  Does it change the base reason for the script or is it just cosmetic / influential.

Reply:
On top of that, it's harder to see if there were any changes to this class.

Reply:
Putting the old name back is no problem. The reason for the change is that the old name wasn't very accurate in describing what the script does.

Reply:
Which is fine, but class name changes have to be controlled else they just break projects.  Given you are changing the logic and function of this class, it might be better to mark the old as obsolete (so you don't inadvertently alter existing projects using it) and create your new class to supercede it in a future release.

You would need to do A/B testing to ensure your updates don't break existing implementations.

Reply:
The rename was 100% about being more accurate about what the class does. The core functionality doesn't change. some logic definitely does (ex: provide a means to fix #1825, and to actually handled disabled influencers).

I am fine using the old name and logging an issue to rename in vNext.